server/jersey-server/src/main/java/com/quorum/tessera/server/JerseyServer.java

```diff
@@ -76,7 +77,12 @@ public void start() throws Exception {
         final ResourceConfig config = ResourceConfig.forApplication(application);
 
         config.addProperties(initParams)
-                .register(MetricsResource.class);
+            .register(MetricsResource.class)
+            .register(LoggingFilter.class);
+
+        if (serverConfig.getCrossDomainConfig() != null) {
```

Comment:
Check if it is unix socket and don't add it... Then you can remove the check in the filter...